libs/sdk-model/api/sdk-model.api.md

```diff
@@ -799,6 +799,7 @@ export type IDashboardFilterReference = IDashboardDateFilterReference | IDashboa
 
 // @alpha
 export interface IDashboardLayout<TWidget = IDashboardWidget> {
+    activeHeaderIndex?: number | null;
```

Comment:
This seems as a wrong place for this, layout is stored on the backend side.
If I understand it correctly this is needed only for the runtime and only in Dashboard component.